packages/network-controller/src/rpc-service/rpc-service.ts

```diff
@@ -566,6 +579,12 @@ export class RpcService implements AbstractRpcService {
         throw rpcErrors.parse({
           message: 'RPC endpoint did not return JSON.',
         });
+      } else if (error instanceof BrokenCircuitError) {
+        this.#logger?.info(error);
```

Comment:
This doesn't seem to show anything in the extension background DevTools. Not sure if we need to use `debug` or if logging just doesn't work like we expect.

Reply:
I wasn't passing the logger down correctly so that's why this didn't work. After fixing that I was able to successfully verify that this works. I also ended up changing this to `warn` to ensure that a stack trace is show in the logs.